bruno/collections/Rafiki/Rafiki Admin APIs/Cancel Outgoing Payment.bru

```diff
@@ -22,11 +22,8 @@ body:graphql {
         quote {
           createdAt
           expiresAt
-          highEstimatedExchangeRate
           id
-          lowEstimatedExchangeRate
-          maxPacketAmount
-          minExchangeRate
+          estimatedExchangeRate
```

Comment:
we also need to update the `Examples` folder (2 `Create Quote` requests in there)

Reply:
Nice catch. I removed everything from bruno dir.